### PR TITLE
Project Clean up

### PR DESCRIPTION
**Pull request by sdorre**
* Remove useless files
* Clean formatting
* Use tab instead of spaces
* Use tag to identify Zephyr version
* Move some files

### CODE REVIEW

File: app/hypnos/prj.conf [hunk 1]
```diff
@@ -49,3 +49,5 @@ CONFIG_BT_ATT_ENFORCE_FLOW=n
 
 # Power management
 CONFIG_DEVICE_POWER_MANAGEMENT=y
+CONFIG_SYS_PM_STATE_LOCK=y
+CONFIG_SYS_POWER_DEEP_SLEEP_STATES=y
```
Comment (endian-albin) [inline app/hypnos/prj.conf:53]: Remove these two. They don't belong to a cleanup PR.

File: app/hypnos/src/battery.c [hunk 2]
```diff
@@ -131,12 +131,12 @@ void battery_show_status()
 {
 	battery_update_percentage();
 	if (charging) {
-		LOG_INF("Battery status: %u%% (charging)",
-			percentage);
+		// LOG_INF("Battery status: %u%% (charging)",
+			// percentage);
```
Comment (endian-albin) [inline app/hypnos/src/battery.c:135]: Either remove these lines completely or keep them uncommented.

File: app/hypnos/src/battery.c [hunk 3]
```diff
@@ -131,12 +131,12 @@ void battery_show_status()
...
 		gfx_battery_set_label(BAT_CHARGE);
 	} else {
-		LOG_INF("Battery status: %u%% (discharging)",
-			percentage);
+		// LOG_INF("Battery status: %u%% (discharging)",
+			// percentage);
```
Comment (endian-albin) [inline app/hypnos/src/battery.c:139]: Either remove these lines completely or keep them uncommented.

File: app/hypnos/src/clock.c [hunk 4]
```diff
@@ -87,5 +87,5 @@ void clock_show_time()
 	strftime(date_label_str, 32, "%a %d %b", &ti);
 	gfx_time_set_label(time_label_str);
 	gfx_date_set_label(date_label_str);
-	LOG_INF("%s | %s", log_strdup(date_label_str), log_strdup(time_label_str));
+	// LOG_INF("%s | %s", log_strdup(date_label_str), log_strdup(time_label_str));
```
Comment (endian-albin) [inline app/hypnos/src/clock.c:90]: Either remove completely or keep it uncommented.

File: app/hypnos/src/main.c [hunk 5]
```diff
@@ -18,6 +18,8 @@
 #include "gfx.h"
 #include "log.h"
 
+#include <power/power.h>
+
```
Comment (endian-albin) [inline app/hypnos/src/main.c:22]: Remove: doesn't belong to this PR

File: app/hypnos/src/main.c [hunk 6]
```diff
@@ -26,13 +28,15 @@ void bt_thread(void);
 #define PRIORITY 7
 
 K_THREAD_DEFINE(bt_id, STACKSIZE, bt_thread, NULL, NULL, NULL,
-                PRIORITY, 0, 0);
+			PRIORITY, 0, 0);
```
Comment (endian-albin) [inline app/hypnos/src/main.c:31]: This makes indentation wrong. I've got Emacs configured to use Linux C style and that puts PRIORITY just below bt_id.

File: app/hypnos/src/main.c [hunk 7]
```diff
@@ -26,13 +28,15 @@ void bt_thread(void);
...
 K_THREAD_DEFINE(main_id, STACKSIZE, main_thread, NULL, NULL, NULL,
-                PRIORITY, 0, 0);
+			PRIORITY, 0, 0);
```
Comment (endian-albin) [inline app/hypnos/src/main.c:33]: See above

File: app/hypnos/src/main.c [hunk 8]
```diff
@@ -52,7 +56,6 @@ void main_thread(void)
 		LOG_INF("Disabling BLE advertising...");
 		bt_adv_stop();
 		cts_sync_enable(false);
-		k_cpu_idle();
```
Comment (endian-albin) [inline app/hypnos/src/main.c:55]: This doesn't belong to this PR. Please do any power-related changes in a separate branch. I've already measured that k_cpu_idle() does makes a different in power consumption so I don't want to see it removed without motivation.

File: app/hypnos/src/main.c [hunk 9]
```diff
@@ -68,6 +71,5 @@ void bt_thread(void)
 			cts_sync_init();
 		}
 		cts_sync_enable(true);
-		k_cpu_idle();
```
Comment (endian-albin) [inline app/hypnos/src/main.c:71]: Again, do this in a separate branch with a motivation.

**Review by endian-albin — CHANGES_REQUESTED** [review]
Nice work. I found just some minor issues. Please do the power management changes in a separate PR with motivation.

File: app/hypnos/prj.conf (hunk 1 above)
Comment (sdorre) [reply app/hypnos/prj.conf]: woups. Sunday morning is not the best time to think straight. lol  All fixed.

File: app/hypnos/src/bt.c [hunk 10]
```diff
@@ -45,9 +45,9 @@ struct k_sem enable_bt_sem;
 struct k_sem disable_bt_sem;
 
 static const struct bt_data ad[] = {
-        BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
-        BT_DATA_BYTES(BT_DATA_UUID16_ALL,
-		      0x0d, 0x18, 0x0f, 0x18, 0x05, 0x18),
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	BT_DATA_BYTES(BT_DATA_UUID16_ALL,
+		0x0d, 0x18, 0x0f, 0x18, 0x05, 0x18),
```
Comment (endian-albin) [inline app/hypnos/src/bt.c:50]: Move this up to the previous line. It will not be too long.

File: app/hypnos/src/event_handler.c [hunk 11]
```diff
@@ -47,10 +47,10 @@ static struct sensor_trigger tap = {
...
 {
 	/* Initialize GPIOs */
-        charging_dev = device_get_binding("GPIO_0");
-        gpio_pin_configure(charging_dev, BAT_CHA, GPIO_INPUT | GPIO_INT_EDGE_BOTH);
-        gpio_init_callback(&charging_cb, battery_charging_isr,
-                BIT(BAT_CHA));
+	charging_dev = device_get_binding("GPIO_0");
+	gpio_pin_configure(charging_dev, BAT_CHA, GPIO_INPUT | GPIO_INT_EDGE_BOTH);
+	gpio_init_callback(&charging_cb, battery_charging_isr,
+			BIT(BAT_CHA));
```
Comment (endian-albin) [inline app/hypnos/src/event_handler.c:53]: Move up to the previous line.

File: app/hypnos/src/main.c [hunk 12]
```diff
@@ -26,13 +26,15 @@ void bt_thread(void);
...
-                PRIORITY, 0, 0);
+		PRIORITY, 0, 0);
 K_THREAD_DEFINE(main_id, STACKSIZE, main_thread, NULL, NULL, NULL,
-                PRIORITY, 0, 0);
+		PRIORITY, 0, 0);
 
 /* ******** Functions ******** */
 void main(void)
 {
+	sys_pm_ctrl_disable_state(SYS_POWER_STATE_DEEP_SLEEP_1);
```
Comment (endian-albin) [inline app/hypnos/src/main.c:36]: Remove

**Review by endian-albin — CHANGES_REQUESTED** [review]
I found a few more things that need fixing.

**Comment by endian-albin** [comment]
Just a general comment regarding indentation: yes, tabs should be used instead of spaces, in general, but **spaces** may be added to tabs to align the code segments nicely, like this:
```
LONG_MACRO_MONSTER(abc, def, ghi, jkl, mno....
<--tab-><--tab->███xyz)
```

**Comment by endian-albin** [comment]
Thanks, now just rebase on master, since we merged the new touch sensor stuff, and fix any conflicts.

**Review by endian-albin — APPROVED** [review]
